safe_transaction_service/history/indexers/tx_processor.py

```diff
@@ -603,7 +603,7 @@ def __process_decoded_transaction(
                 SafeRelevantTransaction.objects.get_or_create(
                     ethereum_tx=ethereum_tx,
                     safe=contract_address,
-                    defaults={"timestamp": ethereum_tx.created},
+                    defaults={"timestamp": ethereum_tx.block.timestamp},
```

Comment:
It is the related field included in the queryset ethereum_tx?
If not this will create a new request to database for each ethereum_tx.